lstchain/reco/r0_to_dl1.py

```diff
@@ -448,9 +448,13 @@ def r0_to_dl1(
 
     # initialize the writer of the interleaved events 
     interleaved_writer = None
-    if 'write_interleaved_events' in config:
+    if 'write_interleaved_events' in config and not is_simu:
         interleaved_writer_config = Config(config['write_interleaved_events'])
         dir, name = os.path.split(output_filename)
+
+        # create output dir in the data-tree if necessary
```

Comment:
this interleaved dir creation is already done by the script that uses this function. It doesn't hurt to leave this here, but perhaps remove it to avoid duplication.